torch_ort_inference/requirements.txt

```diff
@@ -3,5 +3,5 @@
 # In an environment where a requirement is already met, the package will not be updated.
 # If a package of a specific version is needed, the package can be installed
 # with version being specified (see Readme.md).
-#onnxruntime-ortinference-openvino>=1.12.0
+onnxruntime-openvino>=1.11.0
 torch>=1.8
```

Comment:
Since we have not validated versions <1.11 and the versions <1.12 may not contain Aten fallback, it is better if we make 1.12 as the minimum version.

Reply:
Noted. PR updated.